lib/commands/moveToActive-8.lua

```diff
@@ -50,17 +51,22 @@ if jobId then
 
   if(maxJobs) then
     local rateLimiterKey = KEYS[6];
+
     -- local jobCounter = tonumber(rcall("GET", rateLimiterKey))
-    local jobCounter = tonumber(rcall("INCR", rateLimiterKey))
+    if(ARGV[9]) then
+      local handle = string.match(jobId, "[^:]+$");
```

Comment:
handle could be nil if jobId for some reason does not comply with the naming convention. I propose we check for nil and in that case no rateLimiting is performed at all.

Reply:
@manast thanks for the code review and useful comments. I have updated the code to cover those cases.